File: tests/expected/print.go
package main

import (
	"fmt"
)

func Show() {
	fmt.Printf("%v\n", "b")
	fmt.Printf("%v %v\n", 2, "b")
	var a float64 = 2.1
	fmt.Printf("%v\n", a)
	var b float64 = 2.1
	fmt.Printf("%v\n", b)
	var c bool = true
	if c {
		fmt.Printf("%v\n", "True")
	} else {
		fmt.Printf("%v\n", "False")
	}
}

func main() {
	Show()
}

File: tests/expected/print.zig
const std = @import("std");

fn print(comptime fmt: []const u8, args: anytype) void {
    const io = std.Io.Threaded.global_single_threaded.io();
    var buffer: [1024]u8 = undefined;
    var writer = std.Io.File.stdout().writerStreaming(io, &buffer);
    const out = &writer.interface;
    out.print(fmt, args) catch return;
    out.flush() catch return;
}
pub fn show() void {
    print("{s}\n", .{"b"});
    print("{} {s}\n", .{ 2, "b" });
    const a: f64 = 2.1;
    print("{}\n", .{a});
    const b: f64 = 2.1;
    print("{}\n", .{b});
    const c: bool = true;
    print("{s}\n", .{if (c) "True" else "False"});
}

pub fn main() void {
    show();
}
